docs/querying/basics.md

```diff
@@ -169,6 +169,20 @@ The same works for range vectors. This returns the 5-minutes rate that
 
     rate(http_requests_total[5m] offset 1w)
 
+## Subquery
+
+Subquery allows you to run an instant query for a given range and resolution. Like how range vector selector is for vector selector, subquery is for an instant query. The result of a subquery is a matrix. 
```

Comment:
This comparison to range vector selectors is confusing. It'd be instant vector expressions.